norddrop/src/device/mod.rs

```diff
@@ -665,12 +669,12 @@ fn open_database(
                     Ok(storage) => Ok(storage),
                     Err(err) => {
                         let error = ffi::types::NORDDROP_RES_DB_ERROR;
-                        moose.developer_exception(
-                            error as i32,
-                            err.to_string(),
-                            "Failed to open DB after cleanup".to_string(),
-                            "DB Error".to_string(),
-                        );
+                        moose.developer_exception(DeveloperExceptionEventData {
```

Comment:
Maybe the `DeveloperExceptionEventData` should contain `&str`? Since most of the things here are static strings

Reply:
the moose functions still expect a `String` in the end, while this would technically be an optimization when using the other moose implementations, i chose to keep it consistent with the other structs

Reply:
Ok. I get your point since real moose is used in the prod, we should optimize for it